Tools/scripts/summarize_stats.py

```diff
@@ -120,6 +123,26 @@ def load_raw_data(input: Path) -> RawData:
 def save_raw_data(data: RawData, json_output: TextIO):
     json.dump(data, json_output)
 
+@functools.cache
+def _get_uop_flags_from_file(
+    flag_names: tuple[str] = None,
+    filepath: str | Path = "Include/internal/pycore_uop_metadata.h",
+) -> dict[str, list[str]]:
+    flags = {}
+    with open(SOURCE_DIR / filepath) as spec_src:
+        for line in spec_src:
+            if line.startswith("    [_"):
+                if name :=  re.search(r"\[[_A-Z0-9]+\]", line):
+                    uop = name.group().strip("[]")
```

Comment:
```suggestion
                    uop = name.group().removesuffix("[]")
```

Reply:
These lines look like `    [_STORE_SUBSCR_LIST_INT] = HAS_DEOPT_FLAG,` - this is to strip the [] around the name of the uop. Perhaps I'll just change the regex to only capture the Uop name in the first place.

Reply:
Change made in e9fc84d5.